draft-ietf-ppm-dap.md

```diff
@@ -654,13 +655,19 @@ configuration of each aggregator. See {{compliance}} for information on HPKE
 algorithm choices.
 
 Clients retrieve the HPKE configuration from each aggregator by
-sending an HTTP GET request to `[aggregator]/hpke_config?task_id=[task-id]`, where
+sending an HTTP GET request to `[aggregator]/hpke_config`, where
 `[aggregator]` is the aggregator's endpoint URL, obtained from the task
-parameters, and `[task-id]` is the task ID obtained from the task parameters,
+parameters. A deployment may choose to use different HPKE configurations
+for different tasks, so clients MAY specify a query parameter `task_id`
+when sending an HTTP GET request to `[aggregator]/hpke_config?task_id=[task-id]`,
+where `[task-id]` is the task ID obtained from the task parameters,
 encoded in Base 64 with URL and filename safe alphabet with no padding, as
 specified in sections 5 and 3.2 of {{!RFC4648}}. If the aggregator does not
 recognize the task ID, then it responds with HTTP status code 404 Not Found and
-an error of type `unrecognizedTask`. The aggregator responds to well-formed
+an error of type `unrecognizedTask`. If a deployment enforces different HPKE
+configurations for different tasks, and a client doesn't specify `task_id` in the
+HTTP GET request, then the aggregator responds with HTTP status code 400 Bad Request
```

Comment:
```suggestion
HTTP GET request, then the aggregator MUST respond with HTTP status code 400 Bad Request
```

Reply:
This could be stated a bit simpler as:
> then the Aggregator MUST abort with error `missingTaskId`.

Reply:
I added the MUST keyword, but still kept the HTTP status code, to be more explicit.

Reply:
That's fine, but if later on we decide to change the status code, we'll have to update here as well. That's why it's better to specify something once.